inst/emails/contact_aon_recurring

```diff
@@ -1,12 +1,12 @@
 Dear %s,
 
-The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
+The NSF-funded Arctic Data Center support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
 
-If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
+Submissions to the Arctic Data Centre can be done here: https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
 
-Please be sure to submit your data and metadata to the Arctic Data Center sufficiently before your report is due to allow adequate processing time for us to review your submission and for any needed revisions to be prepared.  While we strive to process data as quickly as possible, larger data sets can take significantly more time to process, so please plan accordingly.  Also, there will be less need for multiple rounds of revisions if your initial submission uses open, non-proprietary data formats, includes data that is well-organized, and includes high quality, detailed documentation.  For example, you should document all aspects of your data, including detailed methodological descriptions and structural information about the contents of your data.  See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations.
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. Please take this into account when planning a submission for your report.
```

Comment:
Some of the detail in this paragraph is useful.  At a minimum, I think we should retain the "See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations." Why was this paragraph removed?